src/ekore/operator_matrix_elements/polarized/space_like/as1.py

```diff
@@ -0,0 +1,112 @@
+r"""
+This module contains the |NLO| |OME| (OMEs) in the polarized case for the matching conditions in the |VFNS|.
+Heavy quark contribution for intrinsic evolution are not considered for the polarized case.
+The matching conditions for the |VFNS| at :math:`\mu_F^2 \neq m_H^2` are provided in :cite:`Bierenbaum_2023`. In the paper, the fraction :math:`\mu_F^2 / m_H^2` inside the log is inverted, yielding an additional factor of (-1) wherever L has an odd power. Additionally, a different convention for the anomalous dimensions is used, yielding a factor 2 in the |OME|'s wherever they are present. The anomalous dimensions and beta function with the addition 'hat', have the form :math:`\gamma_hat = gamma(nf+1) - gamma(nf)`.
+"""
+import numba as nb
+import numpy as np
+
+from eko.constants import TR
+
+from ....anomalous_dimensions.polarized.space_like.as1 import gamma_qg as gamma0_qg
+from ...unpolarized.space_like.as1 import A_gg as A_gg_unpol
+
+
+@nb.njit(cache=True)
+def gamma0_qghat(n, nf):
+    r"""Compute the |LO| polarized quark-gluon anomalous dimension with the addition 'hat' and thus, discarding the part proportional 'nf'. The factor 2 is included due to convention mentioned above.
+
+    Parameters
+    ----------
+    N : complex
+      Mellin moment
+    nf : int
+      Number of active flavors
+
+    Returns
+    -------
+    complex
+      |LO| polarized quark-gluon anomalous dimension (hat) :math:`\\\hat{gamma_{qg}}^{(0)}(N)`
+
+    """
+    return 2 * gamma0_qg(n, nf) / nf
```

Comment:
Sorry for being picky:  
here I think it's better to spell out the function explicitly, this way we can drop the nf dependency in all the `as1` module.

Reply:
else you can also pass `nf=1` and avoid the division if the term is fully proportional to `nf`. This would be the cleanest solution.